### PR TITLE
Add support for killing child processes with SIGINT signal

### PR DESCRIPTION
**Pull request by EzzhevNikita**
This PR is a part of solution for [issue](https://github.com/microsoft/azure-pipelines-tasks/issues/11468) in azure-pipelines-tasks repo. 

Customer has a problem with getting of SIGINT signals inside his build script that contains some additional logic when pipeline run cancellation  happens. This PR should add support of sending SIGINT signals to child processes to get them the ability to perform a number of operations before closing the process in SIGINT handler.

### CODE REVIEW

File: node/toolrunner.ts [hunk 1]
```diff
@@ -1022,6 +1023,16 @@ export class ToolRunner extends events.EventEmitter {
         res.stderr = (r.stderr) ? r.stderr.toString() : '';
         return res;
     }
+
+    /**
+     * Used to close child process by sending SIGNINT signal.
+     * It allows to executed script to have some aditional logic on SIGINT, before exiting.
+     */
+    public killChildProcess() {
```
Comment (anatolybolshakov) [inline node/toolrunner.ts:1031]: Nit: could you add ':void' as return type?

Comment (EzzhevNikita) [reply node/toolrunner.ts]: Yeah, thanks, added

**Comment by EzzhevNikita** [comment]
@damccorm could please take a look this pr?

**Review by anatolybolshakov — APPROVED** [review]
Looks good, thanks!

**Review by alexander-smolyakov — APPROVED** [review]
LGTM, thanks!

**Review by damccorm — APPROVED** [review]

**Comment by damccorm** [comment]
Could you bump the version in the package.json and package-lock.json so we can release?

**Comment by EzzhevNikita** [comment]
> Could you bump the version in the package.json and package-lock.json so we can release?

**Comment by EzzhevNikita** [comment]
Sory, hit the wrong button

**Comment by EzzhevNikita** [comment]
@damccorm version bumped

**Comment by damccorm** [comment]
Published!

**Pull request merged**